posts/change-data-capture-cloudsql-postgres-with-apache-airflow-1.md

```diff
@@ -0,0 +1,1002 @@
+            
+        start_export = DummyOperator(
+            task_id='start_export',
+            trigger_rule='one_success'
+        )
+        
+        complete_export = DummyOperator(
+            task_id='complete_export',
+            trigger_rule='all_success'
+        )
+        
+        gcs_schema_keypath = (
+            f"{gcs_root}/"
+            "schemas/"
+            "{{execution_date.year}}/"
+            "{{execution_date.month}}/"
+            "{{execution_date.day}}/"
+            f"{db_name}/"
+            f"{db_name}_schema_"
+            "{{ts_nodash}}.csv"
+        )
+
+        get_schema = CloudSqlCsvExportOperator(
+            task_id = 'get_schema',
+            gcp_conn_id = gcp_conn_id,
+            database_instance = db_instance_id,
+            project_id = gcp_project_id,
+            database_name = db_name,
+            gcs_bucket = gcs_bucket,
+            gcs_key = gcs_schema_keypath,
+            offload = offload_export,
+            params = {
+                "tables": tables
+            },
+            export_sql = 'cloud_sql_cdc/export_csv/get_schema.sql',
+            pool = 'cloudsql_operations',
+            trigger_rule='one_success'
+        )
+         
+        complete_dag = DummyOperator(task_id='complete_dag')
+
+        for table in tables:
+            
+            gcs_raw_keypath = (
+                f"{gcs_root}/"
+                "raw/"
+                f"{table}/"
+                "{{execution_date.year}}/"
+                "{{execution_date.month}}/"
+                "{{execution_date.day}}/"
+                f"{table}_"
+                "{{ts_nodash}}.csv"
+            )
+
+            export_table = CloudSqlCsvExportOperator(
+                task_id = f'export_{table}',
+                gcp_conn_id = gcp_conn_id,
+                database_instance = db_instance_id,
+                project_id = gcp_project_id,
+                database_name = db_name,
+                gcs_bucket = gcs_bucket,
+                gcs_key = gcs_raw_keypath,
+                offload = offload_export,
+                export_sql = f'cloud_sql_cdc/export_csv/{table}.sql',
+                pool = 'cloudsql_operations',
+                trigger_rule='one_success'
+            )
+            get_schema >> export_table >> complete_export  
+            
+        kickoff_dag >> start_export >> get_schema
+        complete_export >> complete_dag
+    ```
+    Now, we have a DAG that extracts change data from our tables at an hourly interval as soon as that interval completes. We can now deploy this to production.
+
+## Step 14: Deploy to Production
+
+Once you commit your changes to git and merge to branches as needed, you are ready to deploy this to production.
+
+### **Create an Airflow Deployment**
+
+To create an Airflow Deployment on Astronomer, log into [Astronomer Cloud](https://app.gcp0001.us-east4.astronomer.io/), open your Workspace, and click **New Deployment**.
```

Comment:
Change this link and language up a bit- we currently don't allow self-service for new signups.

"To create an Airflow Deployment on Astronomer, start a free trial of [Astronomer Cloud](https://astronomer.io/get-astronomer)..."

Reply:
^that link is to https://astronomer.io/get-astronomer

Reply:
These are not my words. I took it from the other page https://www.astronomer.io/docs/cloud/stable/deploy/deploy-cli#step-1-create-an-airflow-deployment

This is a good use case for markdown modules. Then we can change one and have it update everywhere.

Reply:
Feel free to change this one or resolve as needed. I don't know

Reply:
I think we just need an additional step telling people to create an Astronomer account. If that's the case, we can appropriate the language from this step of the [Cloud quickstart](https://www.astronomer.io/docs/cloud/stable/get-started/quickstart#step-1-start-your-trial):